src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundledAppWithSubDirs.cs

```diff
@@ -84,15 +79,12 @@ public void Bundle_Framework_dependent_NoBundleEntryPoint()
         }
 
         [InlineData(BundleOptions.None)]
-        [InlineData(BundleOptions.BundleNativeBinaries)]
```

Comment:
I wonder why it's OK to decrease test coverage here. Is it that in 8, all of the framework binaries are always "included" and thus there's nothing to do as there are no "native binaries" on the input really?

Reply:
Yeah, I removed it since there are no native binaries for these tests. I could explicitly add one, but the `BundleExtractToSpecificPath` tests (and one of the `SingleFileApiTets`) still cover the `BundleNativeBinaries` option.

In the existing system we have native binaries because we bundle all the output of a self-contained build and the tests check that `coreclr` gets bundled/extracted. That would be change when they move to this mechanism, but I'd explicitly bundle a native binary in those those tests to keep the coverage - I think that is still be reasonable coverage without having other tests always do the full matrix of options.

Reply:
Thanks for the explanation, makes sense.